manifest.yml

```diff
@@ -8,14 +8,237 @@ output_dir: out
+      # - 'resource_sharing on'
+    features:
+      a_bw_i: 8
+      b_bw_i: 8
+      o_bw: 8
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_8_8_16_bit.sv
+    flags: 
+    features:
+      a_bw_i: 8
+      b_bw_i: 8
+      o_bw: 16
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_8_16_24_bit.sv
+    flags: 
+    features:
+      a_bw_i: 8
+      b_bw_i: 16
+      o_bw: 24
+    
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_16_16_16_bit.sv
+    flags: 
+    features:
+      a_bw_i: 16
+      b_bw_i: 16
+      o_bw: 16
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_16_32_32_bit.sv
+    flags: 
+    features:
+      a_bw_i: 16
+      b_bw_i: 32
+      o_bw: 32
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_16_32_48_bit.sv
+    flags: 
+    features:
+      a_bw_i: 16
+      b_bw_i: 32
+      o_bw: 48
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_18_18_18_bit.sv
+    flags: 
+    features:
+      a_bw_i: 18
+      b_bw_i: 18
+      o_bw: 18
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_18_18_36_bit.sv
+    flags: 
+    features:
+      a_bw_i: 18
+      b_bw_i: 18
+      o_bw: 36
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_18_25_43_bit.sv
+    flags: 
+    features:
+      a_bw_i: 18
+      b_bw_i: 25
+      o_bw: 43
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_32_32_32_bit.sv
+    flags: 
+    features:
+      a_bw_i: 32
+      b_bw_i: 32
+      o_bw: 32
+    
+    # Filepaths are assumed to be relative to project root.
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_32_32_64_bit.sv
+    flags: 
+    features:
+      a_bw_i: 32
+      b_bw_i: 32
+      o_bw: 64
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_32_48_80_bit.sv
+    flags: 
+    features:
+      a_bw_i: 32
+      b_bw_i: 48
+      o_bw: 80
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_32_64_96_bit.sv
+    flags: 
+    features:
+      a_bw_i: 32
+      b_bw_i: 64
+      o_bw: 96
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_64_64_64_bit.sv
+    flags: 
+    features:
+      a_bw_i: 64
+      b_bw_i: 64
+      o_bw: 64
+
+  - benchmark:
+    filepath: benchmarks/mul/0_stage/mul_0_stage_unsigned_64_64_128_bit.sv
+    flags: 
+    features:
+      a_bw_i: 64
+      b_bw_i: 64
+      o_bw: 128
+
+  - benchmark:
+    filepath: benchmarks/mul/1_stage/mul_1_stage_unsigned_8_16_24_bit.sv
+    flags: 
+    features:
+      a_bw_i: 8
+      b_bw_i: 16
+      o_bw: 24
+
+  - benchmark:
+    filepath: benchmarks/mul/2_stage/mul_2_stage_unsigned_8_16_24_bit.sv
+    flags: 
+    features:
+      a_bw_i: 8
+      b_bw_i: 16
+      o_bw: 24
     
-  - # Filepaths are assumed to be relative to project root.
```

Comment:
Can you add this comment back in to the first entry in this list? That first entry will serve as an example that we can document, to explain the format of manifest.yml.

Reply:
Sure! I think I had it somewhere way down in `manifest.yml`. I've inserted the comment `# Filepaths are assumed to be relative to project root.` to line 14, directly above `filepath` for the first benchmark entry in benchmarks.